modules/oaipmh/src/main/java/org/opencastproject/oaipmh/server/OaiPmhServer.java

```diff
@@ -236,7 +242,6 @@ String getRepositoryUrl() {
   @Override
   public void destroy() {
     super.destroy();
-    tryUnregisterServlet();
```

Comment:
If we remove this, we can also completely drop this method, can't we?

Reply:
Ups, the method is pretty useless.